06_puppy/sirigithub/stores_test.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type storesSuite struct {
+	suite.Suite
+	store     Storer
+	newStorer func() Storer
+	puppy1    Puppy
+}
+
+func (suite *storesSuite) SetupTest() {
+	suite.store = suite.newStorer()
+	suite.puppy1 = Puppy{ID: 1, Breed: "cavalier", Color: "White", Value: "$1700"}
+	er := suite.store.CreatePuppy(&suite.puppy1)
+	if er != nil {
+		suite.FailNow("Test setup failed to create puppy", er)
+	}
+
```

Comment:
Unnecessary blank line.
No need for blank lines at beginning or end of block.
Use blank lines between type, method and function definitions and within a block for logical separation.

(below as well)

Reply:
Noted. Fixed.